### PR TITLE
List desired cloud provider git repositories as subprojects

### PR DESCRIPTION
**Pull request by calebamiles**
This is a statement of intent to create new Git repositories for the
cloud providers to extract from `kubernetes/kubernetes`. Handling
repositories for newly donated cloud providers will be handled
separately

### CODE REVIEW

**Comment by calebamiles** [comment]
**cc: @kubernetes/sig-architecture-feature-requests**

**Comment by jdumars** [comment]
I don't think it affects this PR, but there was discussion about these repos being more than cloud provider code - meaning volume and other "implementation-specific" components.  There wasn't a good suggestion about what to call it that was generic enough, but perhaps @cheftako @jagosan or @wlan0 can weigh in on that.  Otherwise, LGTM

File: sigs.yaml [hunk 1]
```diff
@@ -1663,6 +1663,31 @@ workinggroups:
...
+      - https://raw.githubusercontent.com/kubernetes/aws-cloud-provider/master/OWNERS
+    - name: azure-cloud-provider
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/azure-cloud-provider/master/OWNERS
+    - name: cloudstack-cloud-provider
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/cloudstack-cloud-provider/master/OWNERS
+    - name: gce-cloud-provider
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/gce-cloud-provider/master/OWNERS
```
Comment (mikedanese) [inline sigs.yaml]: gcp cloud provider instead of gce?

Comment (idvoretskyi) [reply sigs.yaml]: +1. It should be GCP (based on the existing names conventions).

File: sigs.yaml [hunk 2]
```diff
@@ -1663,6 +1663,31 @@ workinggroups:
...
+    - name: openstack-cloud-provider
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/openstack-cloud-provider/master/OWNERS
+    - name: ovirt-cloud-provider
```
Comment (idvoretskyi) [inline sigs.yaml]: Do we have an existing actively supported piece of codebase for ovirt?

Comment (calebamiles) [reply sigs.yaml]: https://github.com/kubernetes/kubernetes/tree/master/pkg/cloudprovider/providers/ovirt

**Comment by calebamiles** [comment]
changed, @mikedanese

**Comment by calebamiles** [comment]
I'm not sure if the script properly handles changes to the template, @jdumars. Since I _definitely_ didn't edit any `README.md` files by hand I was going to ignore the failure

**Comment by jdumars** [comment]
Ok, with non-submitter approval, and 2 legit LGTMs, removing hold and merging.

**Pull request merged**